app/services/DataCreatorService.service.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+  'use strict';
+
+  class DataCreatorService {
+    createData(sortedData, colors) {
+      let i = 0;
+      let data = {
+        bar: [],
+        map: [],
+        linear: []
+      }
+      Object.entries(sortedData).forEach(([key, value]) => {
+        data.bar = this.createBarData(data.bar, key, value, colors[i]);
+        data.map = this.createMapData(data.map, key, value, colors[i]);
+        data.linear = this.createLinearData(data.linear, key, value, colors[i]);
+        i++;
+      });
+      return data;
+    }
+    createBarData(bar, key, value, color) {
+      //create JSON for Bar Graph {'word': 'text', 'freq': stream.length, 'color': 'color'}
+      bar.push({word: key, freq: value.length, color: color});
+      return bar;
+    }
+    createMapData(map, key, value, color) {
+      // create JSON for Map Graph {'word': 'text', 'coords': [lon, lat], 'color': 'color'}
+      value.forEach((twit) => {
+        if (twit.coordinates) {
+          return map.push({word: key, coords: twit.coordinates.coordinates, color: color});

```

Comment:
kilka punktow wyjscia to zlo. juz pisalem o tym. 
tutaj zmienia sie tylko coords, wiec moze wrzuc wartosci w let coords = '...'; a pozniej zrob tylko raz returna?